tasks/process_flakes.py

```diff
@@ -23,76 +28,124 @@ class ProcessFlakesTask(BaseCodecovTask, name=process_flakes_task_name):
 
     def run_impl(
         self,
-        _db_session,
+        _db_session: Any,
         *,
-        repo_id,
-        commit_id_list,
-        branch,
-        **kwargs,
+        repo_id: int,
+        commit_id: str,
+        **kwargs: Any,
     ):
-        repo_id = int(repo_id)
+        """
+        this task wants to iterate through uploads for a given commit that have yet to be
```

Comment:
nit: Capitalization of sentences (this and below)

```suggestion
        This task wants to iterate through uploads for a given commit that have yet to be
```